content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -572,6 +572,30 @@ ConfigMap updates.
 
 {{% capture discussion %}}
 
+## Auto-created ConfigMaps
+
+If BoundServiceAccountTokenVolume was enabled(requires v1.13 or later), kube-controller-manager would create ConfigMap `kube-root-ca.crt` in each namespace.
+
+```shell
+kubectl get cm
+NAME               DATA   AGE
```

Comment:
According to the style guide, we should [separate commands from a sample of their output](https://kubernetes.io/docs/contribute/style/style-guide/#separate-commands-from-output).

Does that make sense here?